foreman/Dockerfile

```diff
@@ -13,6 +13,12 @@ COPY data_models/dist/data-refinery-models-* data_models/
 RUN pip install \
     data_models/$(ls data_models -1 | sort --version-sort | tail -1)
 
+COPY common/dist/data-refinery-common-* common/
```

Comment:
The only file in there currently is `data-refinery-common-0.1.0.tar.gz`, however once I actually get to a good point I'll start being diligent about using semantic versioning for every change so I think there'll pretty much be the same file but with different version numbers.